.github/workflows/ci_frontend.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests when there is a pull request
+# with changes within the "frontend" directory 
+
+name: CI_FRONTEND
+
+on:
+  pull_request:
```

Comment:
I was wondering if we needed to trigger these workflows on the `push` event to `master` as well, since it will help show a nice green tick or a red cross on the commit log of the master branch... then I realised that it will all be green ticks anyway since we require the CI to pass before merging.

Semantic merge conflicts will be detected by the CI since the `pull_request` event sends the *merged* commit, not the pull request's branch head.

(Sorry I'm just talking to myself)